option optimize_for = LITE_RUNTIME;

message LegionStickyMessageRequest
{
 required uint32 message_id = 1;
}
